src/game/SpellEffects.cpp

```diff
@@ -1149,6 +1149,13 @@ void Spell::EffectDummy(SpellEffectIndex eff_idx)
 
                     return;
                 }
+                case 32027:                                 // Expedition Flare
+                {
+                    if (!m_CastItem)                        // 32029 = Expedition Preserver | 32030 = Expedition Scout
```

Comment:
Actually I was refering to the other spell effect dummies which have the (m_CastItem) check which use m_CastItem exactly like it was used here, not as the target.

I am just wondering if some spells may rely on actually having the m_CastItem passed along; why else would we be doing it at all if not?